improvement: S3UTILS-115 replication verification util

naren-rajendran opened the pull request.

Commit:
improvement: S3UTILS-115 replication verification util

bert-e commented:
# Hello naren-rajendran,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/s3utils/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue S3UTILS-115 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `1.13.8`


Please check the `Fix Version/s` of S3UTILS-115, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

williamlardier commented on VerifyReplication/index.js:
```diff
@@ -0,0 +1,176 @@
+Usage:
+    node VerifyReplication/index.js
+
+Mandatory environment variables:
```
I do see an improvement that might be useful for large buckets, in case the tool stops for any reason: why don't we periodically store the progress of workers, so that we can either start or restart at any point from a specific object in the original bucket?
What do you think?

naren-rajendran replied:
@williamlardier yes its on my to-do list, it would certainly be an improvement. The current iteration is for a preview (not final).

naren-rajendran replied:
It is adding more complexity than originally visioned. It can be implemented as an improvement, if needed.

Commit:
improvement S3UTILS-115 update constants

Commit:
improvement: S3UTILS-115 enable/disable client level logs

Commit:
improvement: S3UTILS-115 update README.md

Commit:
improvement: S3UTILS:115 corrections

modified variable names of log entries, added last-modified to the log
entries and updated readme to reflect the updated variable names and
dates.

naren-rajendran commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/1.13`



The following branches will **NOT** be impacted:

* `development/1.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/1.13`



The following branches have **NOT** changed:

* `development/1.4`



Please check the status of the associated issue S3UTILS-115.

Goodbye naren-rajendran.
